tldw_Server_API/Config_Files/.env.example

```
@@ -93,6 +93,18 @@ ELEVENLABS_API_KEY=
 CUSTOM_OPENAI_API_KEY=${OPENAI_API_KEY}
 CUSTOM_OPENAI2_API_KEY=${OPENAI_API_KEY}
 CUSTOM_OPENAI_API_2_API_KEY=${OPENAI_API_KEY}
+# Endpoint base URL overrides. CUSTOM_OPENAI_API_BASE is accepted as an alias
+# for OpenAI-compatible naming; CUSTOM_OPENAI_API_IP_1/_2 remain supported.
+CUSTOM_OPENAI_API_IP=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CUSTOM_OPENAI_API_IP key should go before the CUSTOM_OPENAI_API_KEY key

(UnorderedKey)